using_images/other_images/jenkins.adoc

```diff
@@ -75,10 +153,50 @@ The Jenkins password can be configured with the following environment variable:
 |Variable name |Description
 
 |`*JENKINS_PASSWORD*`
-|Password for the `*admin*` user.
+|Password for the `*admin*` user when using default Jenkins authentication.
+
+|`*OPENSHIFT_ENABLE_OAUTH*`
+|Determines whether the OpenShift Login plugin manages authentication when logging into Jenkins.  Enabled when set to any non-empty value other than "false".
+
+|`*OPENSHIFT_PERMISSIONS_POLL_INTERVAL*`
+|Specifies in seconds how often the OpenShift Login plugin polls {product-title} for the permissions associated with each user defined in Jenkins.
 
 |===
 
+=== Cross Project Access
+
+If you are going to run Jenkins somewhere other than as a deployment within your same project you will need to provide an access token to Jenkins to access your project.
+
+    1. Identify the service account token secret:
+
+            $ oc describe serviceaccount default
+
+        Example output:
+
+            Name:       default
+            Labels:     <none>
+            Secrets:    {  default-token-uyswp    }
+                        {  default-dockercfg-xcr3d    }
+            Tokens:     default-token-izv1u
+                        default-token-uyswp
+
+        In this case the secret name is `default-token-uyswp`
+
+    2. Retrieve the token from the secret:
+
+            $ oc describe secret <secret name from above> # e.g. default-token-izv1u
+
+        Example output:
+
+            Name:       default-token-izv1u
+            Labels:     <none>
+            Annotations:    kubernetes.io/service-account.name=default,kubernetes.io/service-account.uid=32f5b661-2a8f-11e5-9528-3c970e3bf0b7
+            Type:   kubernetes.io/service-account-token
+            Data
+            ====
+            ca.crt: 1066 bytes
+            token:  eyJhbGc..<content cut>....wRA
```

Comment:
"The token field contains the token value Jenkins will need to access the project."